packaging/plf-community-tomcat-standalone/src/main/catalog/meeds-distrib-catalog.json

```diff
@@ -0,0 +1,115 @@
+[
+  {
+    "id": "meeds-app-center",
+    "version": "${addon.exo.app-center.version}",
+    "unstable": true,
+    "name": "App Center",
+    "description": "An addon to manage displayed user applications in menu",
+    "sourceUrl": "https://github.com/meeds-io/app-center",
+    "downloadUrl": "http://addons.exoplatform.org/n/exo-addons-snapshots/org.exoplatform.addons.app-center/app-center-packaging/${addon.exo.app-center.version}/zip",
```

Comment:
These links will work only for snapshot version, Other versions ( milestone will be impacted )

Reply:
Ah, yes, good point @hbenali . Which permanent URL to use ?

Reply:
Is it of format:
` https://repository.exoplatform.org/service/local/artifact/maven/redirect?r=exo-jboss-snapshots&g=org.exoplatform.jcr&a=jcr-packaging&v=${addon.exo.jcr.version}&e=zip`
I think the name of repository is mandatory here as well, isn't it ?

Reply:
Ah, found it, we can use `public` proxy repository. I will test it, but will it access to `staging` repository too ?

Reply:
Fixed cc @hbenali